MyApplication/app/src/main/java/com/jonathanlieblich/myapplication/Helper.java

```diff
@@ -0,0 +1,182 @@
+        sqLiteDatabase.execSQL("DROP TABLE IF EXISTS "+TABLE_JOB);
+        onCreate(sqLiteDatabase);
+    }
+
+    public void addEmployee(Employee employee) {
+        SQLiteDatabase db = getWritableDatabase();
+        ContentValues values = new ContentValues();
+        values.put(COL_FIRST_NAME, employee.getFirstName());
+        values.put(COL_LAST_NAME, employee.getLastName());
+        values.put(COL_CITY, employee.getCity());
+        values.put(COL_SSN, employee.getSsn());
+        values.put(COL_BIRTH, employee.getBirthYear());
+        db.insertOrThrow(TABLE_EMPLOYEE, null, values);
+        db.close();
+    }
+
+    public void addJob(Job job) {
+        SQLiteDatabase db = getWritableDatabase();
+        ContentValues values = new ContentValues();
+        values.put(COL_SSN, job.getSsn());
+        values.put(COL_COMPANY, job.getCompany());
+        values.put(COL_SALARY, job.getSalary());
+        values.put(COL_EXP, job.getExperience());
+        db.insertOrThrow(TABLE_JOB, null, values);
+        db.close();
+    }
+
+    public List<String> nameByCompany() {
+        SQLiteDatabase db = getReadableDatabase();
+
+        String query = "SELECT "+COL_FIRST_NAME+","+COL_LAST_NAME+" FROM "+TABLE_EMPLOYEE
+                +" JOIN "+TABLE_JOB+" ON "+TABLE_EMPLOYEE+"."+COL_SSN+" = "+TABLE_JOB+"."+COL_SSN
+                +" WHERE "+COL_COMPANY+" = 'Macys'";
+        Cursor cursor = db.rawQuery(query, null);
+
+        List<String> names = new ArrayList<>();
+
+        if(cursor.moveToFirst()) {
+            while(!cursor.isAfterLast()) {
+                names.add(cursor.getString(cursor.getColumnIndex(COL_FIRST_NAME))
+                +" "+cursor.getString(cursor.getColumnIndex(COL_LAST_NAME)));
+                cursor.moveToNext();
+            }
+            cursor.close();
+            return names;
+        }
+        cursor.close();
+        return null;
+    }
+
+    public List<String> companyByCity() {
+        SQLiteDatabase db = getReadableDatabase();
+
+        String query = "SELECT "+COL_COMPANY+" FROM "+TABLE_EMPLOYEE+" JOIN "+TABLE_JOB+" ON "
+                +TABLE_EMPLOYEE+"."+COL_SSN+" = "+TABLE_JOB+"."+COL_SSN+" WHERE "+COL_CITY +" = 'Boston'";
+        Cursor cursor = db.rawQuery(query, null);
+
+        List<String> companies = new ArrayList<>();
+
+        if(cursor.moveToFirst()) {
+            while(!cursor.isAfterLast()) {
+                companies.add(cursor.getString(cursor.getColumnIndex(COL_COMPANY)));
+                cursor.moveToNext();
+            }
+            cursor.close();
+            return companies;
+        }
+        cursor.close();
+        return null;
+    }
+
+    public List<String> highestPaidEmployee() {
+        SQLiteDatabase db = getReadableDatabase();
+
+        String query = "SELECT "+COL_FIRST_NAME+","+COL_LAST_NAME+" FROM "+TABLE_EMPLOYEE+" JOIN "+
+                TABLE_JOB+" ON "+TABLE_EMPLOYEE+"."+COL_SSN+" = "+TABLE_JOB+"."+COL_SSN+" ORDER BY "+COL_SALARY;
```

Comment:
You'll want to add `DESC` at the end of your `ORDER BY` clause.. By default, it sorts in _ascending_ order from smallest to largest, but you want _descending_ order from largest to smallest in this case.